spec/system/category_topics_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+describe "Catergory topics", type: :system, js: true do
```

Comment:
Just a note but for system tests, we're trying to describe the behaviour of what a user will encounter so it will be better to make this more reflective of what a user will experience. For example, "Viewing topics for a category"